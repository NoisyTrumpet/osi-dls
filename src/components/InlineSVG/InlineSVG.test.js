import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { InlineSVG } from './';

describe('<InlineSVG />', () => {
  test('Should render', () => {
    const wrapper = shallow(<InlineSVG />);
    expect(wrapper.getElement()).toBeTruthy();
    expect(toJson(wrapper)).toMatchSnapshot();
  });

  test('test-inlinesvg div should render', () => {
    const inlineSVGComponent = (
      <InlineSVG
        path="test-path"
        title="test-title"
        description="test-description"
        role="img"
        focus
        className="test-inlinesvg"
      />
    );
    const wrapper = shallow(inlineSVGComponent);
    expect(wrapper.find('.test-inlinesvg').length).toBe(1);
  });
});
