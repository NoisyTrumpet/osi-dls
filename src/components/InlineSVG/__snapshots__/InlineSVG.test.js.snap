// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InlineSVG /> Should render 1`] = `
<InlineSVG
  cacheRequests={true}
  description=""
  focus="false"
  preProcessor={null}
  role=""
  src=""
  title=""
  uniquifyIDs={false}
/>
`;
